src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,75 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import java.time.LocalDateTime;
+
+@Entity
```

Comment:
table annotation missed 
```suggestion
@Entity
@Table(name="movie_sessions")
```